contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaObservableQueue.java

```diff
@@ -68,10 +72,11 @@
     public KafkaObservableQueue(final String queueName, final Configuration config) {
         this.queueName = queueName;  // Topic
         this.pollIntervalInMS = config.getIntProperty("kafka.consumer.pollingInterval", 1000);
-        this.pollTimeoutInMs = config.getIntProperty("kafka.consumer.longPollTimeout", 1000);
+        this.pollTimeoutInMs = Duration.ofMillis(config.getIntProperty("kafka.consumer.longPollTimeout", 1000));
```

Comment:
What is the thousand here?

Reply:
1000 is milliseconds. It's a default value, if not provided in config